### ISSUE TITLE
Detect SSO for When Org has SSO Set Up

### ISSUE THREAD

**Issue by cassidycodes** [kamilkisiela/graphql-hive#3055]
We have SSO set up through Okta, but when a developer finds the hive login page, they are not prompted to log in with SSO. Two related features would be great:
1. Add a "login with Okta" (or other SSO provider) button on the home page when the org has it set up.
2. Detect if an email address should be associated with SSO or not. For example, it is not likely that `user@mycompany.com` will need a distinct Google login from their SSO login. If `user@mycompany.com` tries to log in with any other method, redirect them to the SSO page the org has configured.

**Comment by comp615** [comment]
Adding on / clarifying after setting up SSO for my company too. A few things security wanted to look for in this.

Would love to see:
- **Require SSO** (I think essentially what Cassidy is describing in 2 above)
- **Better support for moving to SSO**. I had signed in and created our org with Google SSO. Then after I added OKTA OIDC support, when I signed in, it made a totally new account for me despite the emails being the same. This was surprising as I'd expect email to be the primary key and it should have just logged me in. (see https://supertokens.com/docs/thirdparty/common-customizations/deduplication/implementing-deduplication)
